### PR TITLE
New mini project #1

### PR DESCRIPTION
**Pull request by Tolokov**
Hi, Rahul!
Thanks for the chance to practice for contributing.

- [x]  Added new python file 'red_and_blue_BUTTONS.py
images

### CODE REVIEW

File: red_and_blue_BUTTONS.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# Dec 13, 2021: 'The Matrix Resurrections' and 'Spider Man: No Way Home' are coming out
```
Comment (iamrahulrnair) [inline red_and_blue_BUTTONS.py]: Hi, could you remove these comments, we dont need that ;-)

**Review by iamrahulrnair — CHANGES_REQUESTED** [review]
Please change the comment and commit it, then i'll merge it , looks good though

File: red_and_blue_BUTTONS.py (hunk 1 above)
Comment (Tolokov) [reply red_and_blue_BUTTONS.py]: comment is deleted :)

**Review by iamrahulrnair — APPROVED** [review]

**Comment by iamrahulrnair** [comment]
Thanks for the contribution, and try to do more cool gui stuffs, this looks intersting ,keep up the work :-)

**Pull request merged**